my-app/src/pages/authentication/Registration.tsx

```diff
@@ -0,0 +1,99 @@
+import React, {useContext} from 'react';
+import {Link} from "react-router-dom";
+import styled from "styled-components";
+import {Avatar, Button, Paper, TextField, Typography} from "@mui/material";
+import {AuthContext} from "./AuthContext";
+
+export const Registration = () => {
+    const logic = useContext(AuthContext)
+    const onClickHandler =()=>{
+        logic.userRegister()
+        logic.inputCheck()
+    }
+
+    return (
+        <PaperWrapper>
+            <TypographyWrapper variant="h5">
+                Account Registration
+            </TypographyWrapper>
+            <FormWrapper>
+                <Avatar sx={{width: 90, height: 90}}/>
+            </FormWrapper>
+            <TextFieldWrapper label="Full Name"
+                              onChange={(e) => logic.setRegisterForm({

```

Comment:
also try to keep logic out of jsx tags. Define it before return statement

Reply:
fixed